shared/src/helpers/date/get-formatted-date/get-formatted-date.ts

```diff
@@ -1,12 +1,15 @@
-import { formatDistance } from 'date-fns';
+import { format, formatDistance } from 'date-fns';
 
-type FormatDate = 'distance';
+type FormatDate = 'distance' | 'time';
```

Comment:
maybe it would be better to declare date format as type straightly make it easier to understand the format used
```suggestion
type FormatDate = 'distance' | 'kk:mm, dd/MM/yyyy';
```